src/main/java/org/project36/qualopt/domain/Study.java

```diff
@@ -53,6 +53,17 @@
                joinColumns = @JoinColumn(name="studies_id", referencedColumnName="id"),
                inverseJoinColumns = @JoinColumn(name="participants_id", referencedColumnName="id"))
     private Set<Participant> participants = new HashSet<>();
+    
+    //Check this before sending email to new comer, only send emails to those that in this list
+    //Use @Transient because the database is not correctly configure for this yet, otherwise use @OneToMany
+    /**@ManyToMany
+    @Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
+    @JoinTable(name = "havent_receive_email",  //Don't delete the "_", when naming tables, keep the "_" in between word
+    	joinColumns = @JoinColumn(name="studies_id", referencedColumnName="id"),
+    	inverseJoinColumns = @JoinColumn(name="participants_id", referencedColumnName="id"))
+    **/
+    @Transient
+    private Set<Participant> uninviatedParticipants = new HashSet<>();
```

Comment:
typo spelling error, should be `uninvitedParticipants` (throughout rest of code)